meson.build

```diff
@@ -330,14 +330,6 @@ message('HAVE_PTRACE: @0@'.format(have_ptrace))
 message('USE_PTRACE_WRAP: @0@'.format(use_ptrace_wrap))
 
 checks_level = get_option('checks_level')
-if checks_level == 9999
```

Comment:
maybe add a check for unsupported values?

Reply:
ok 9fc8db7c258880ed4e2d7a2bb245f40fa5764132 ... if you want to test this, do `meson setup --wipe`